.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+<!-- 
+Please indicate after the # which issue you're closing with this PR, if applicable.
+If the PR closes multiple issues, include "closes" before each one is listed.
+You can also link to other issues if necessary, e.g. "See also #1234".
+
+https://help.github.com/articles/closing-issues-using-keywords
+-->
+Closes #
+
+<!-- 
+Please give a brief overview of what has changed or been added in the PR.
+This can include anything specific the maintainers should be looking for when they review the PR.
+-->
+Changes proposed in this pull request:
+
+-
+-
+
+<!-- To be checked off by reviewers -->
+## Checklist
+
+- [ ] PR has an interpretable title with a prefix (`[ENH]`, `[FIX]`, `[REF]`, `[TST]`, `[CI]`, `[MNT]`, `[INF]`) _(see https://neurobagel.org/contributing/pull_requests for more info)_
+- [ ] PR links to GitHub issue with mention `Closes #XXXX`
+- [ ] Checks pass
```

Comment:
can we also have the "Code formatted" bit from the other template https://github.com/neurobagel/.github/pull/5

Reply:
Leaving this out as it would be part of the checks! :)